src/NzbDrone.Core/Notifications/Join/JoinSettings.cs

```diff
@@ -19,7 +20,10 @@ public class JoinSettings : IProviderConfig
         private static readonly JoinSettingsValidator Validator = new JoinSettingsValidator();
 
         [FieldDefinition(0, Label = "API Key", HelpText = "The API Key from your Join account settings (click Join API button).", HelpLink = "https://joinjoaomgcd.appspot.com/")]
-        public string APIKey { get; set; }
+        public string ApiKey { get; set; }
+
+        [FieldDefinition(1, Label = "Device IDs", HelpText = "Comma separated list of Device IDs you'd like to send notifications.  If unset, all devices will receive notifications.", HelpLink = "https://joinjoaomgcd.appspot.com/")]
```

Comment:
`Comma separated list of Device IDs you'd like to send notifications to. If unset, all devices will receive notifications.`